api/src/main/java/org/pmiops/workbench/utils/mappers/LeonardoMapper.java

```diff
@@ -25,8 +25,23 @@
 
   DataprocConfig toDataprocConfig(LeonardoMachineConfig leonardoMachineConfig);
 
+  @Mapping(
+      target = "cloudService",
+      // I wanted to use constant/defaultValue here but MapStruct will only accept a static string,
+      // not an Enum value. This is a problem because we have 3 definitions of `CloudServiceEnum`
+      // and MapStruct doesn't know that this mapper should use the one from LeonardoMachineConfig
+      // while the mapper for LeonardoGceConfig should use its own `CloudServiceEnum` class.
+      // This `expression` field feels kind of hacky since I'm expressing code as a string but
```

Comment:
Would `@aftermapping` + ignore be cleaner here?

Reply:
Hmm, I went with this because its more concise but I can switch over since it would allow for code inspection.